index.html

```diff
@@ -64,5 +64,183 @@ <h2 class="container job-title">I am a Software developer </h2>
             </ul>
         </div>  
     </section>
+    <!-- Recent work section -->
+
+<section class="recent-works">
+    <div class="first">
+        <h3>My Recent Works</h3>
+        <hr>
+    </div>
+    <div class="recent-work" id="recent-works">
+        <a href="" class="women"><img src="assets/Img_Placeholder.png" alt="women doing yoga"></a></div>
+    <div>
+        <h4>Multi-Post-Stories</h4>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="tech-used">
+            <li class="css">Css</li>
+            <li class="HTML">html</li>
+            <li class="boots">Bootstrap</li>
+            <li class="ruby">Ruby</li>  
+        </ul>
+        <form>
+        <button type="button" >See Project</button></form>
+    </div>
+    <!-- first-Profesional-Art-Printing-Data -->
+    <div class="card-section">
+        <div class="work-card">
+            <div class="card-content flex-container">
+                <h4 class="title">Profesional Art Printing Data</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li class="html">html</li>
+                    <li class="Boot">Bootstrap</li>
+                    <li class="Ruby">Ruby</li>
+                </ul>
+            <div>
+               <button class="project-button" type="button" id="firstTitle">See Project</button>
+            </div>
+        </div>
```

Comment:
- Please align this button properly as it looks like the Figma design, so as you keep to details. 
- Please make use of the proper font family for the text in this section, as it is on the Figma design
##### Figma design 
![Screenshot from 2022-06-04 10-23-23](https://user-images.githubusercontent.com/58771507/171993298-f8f94d27-a857-4349-8d1b-adaf94a8503d.png)

##### Your design 
![Screenshot from 2022-06-04 10-22-55](https://user-images.githubusercontent.com/58771507/171993309-2683d1e4-ddfb-442c-a551-a7a545ca59da.png)

Reply:
thanks so much for sharing the feedback.
I did all of them, please if you want to say some more feedback, please give me optional because the time is less.
thank you so much.